webrtc.html

```diff
@@ -9866,10 +9866,9 @@ <h3>canInsertDTMF algorithm</h3>
         nor <code>"sendonly"</code> return <code>false</code> and abort these steps.</li>
         <li>If <var>sender</var>'s <a>[[\SendEncodings]]</a><code>[0].active</code> is
         <code>false</code> return <code>false</code> and abort these steps.</li>
-        <li>If there is no value of <var>j</var> ranging from 0 to the number of
-        codecs, for which <var>sender</var>'s 
-        <a>[[\SendEncodings]]</a><code>[0].codecs[<var>j</var>].mimetype</code>
-        has the value <code>"audio/telephone-event"</code>, return <code>false</code>
+        <li>If no codec with mimetype <code>"audio/telephone-event"</code> has been
+        negotiated for sending, and which the user agent is currently
+        capable of sending, return <code>false</code>
```

Comment:
Nit: I find it hard to parse the negated *"and"* here. Do we mean *"or the user agent is not currently capable of sending"*? Not sure how to phrase better.

Reply:
I made an attempt with https://github.com/w3c/webrtc-pc/pull/1980/commits/33281cdb3207a57a3d10a6ea9264f6db977d536e.